src/table/toolbar/table-toolbar.component.ts

```diff
@@ -106,6 +106,8 @@ export class TableToolbar {
 		return { CANCEL: this._cancelText.value as string };
 	}
 
+	@Output() cancel = new EventEmitter;
```

Comment:
Huh I guess the original works, but this is a bit more consistent: 

```suggestion
	@Output() cancel = new EventEmitter();
```

Reply:
Done! Sorry I missed them haha and also deleted the double spaces, I don't know what happened with lint there :P